docs/installation/linux.md

```diff
@@ -2,7 +2,7 @@ Package installation instructions
 =================================
 
 Supports [Ubuntu 14.04][u14], [Ubuntu 16.04][u16],
-[CentOS 7][cos], [many Linux distributions (AppImage)][lai], and [macOS 10.11][mac].
+[CentOS 7][cos], [Arch Linux][arch], [many Linux distributions (AppImage)][lai], and [macOS 10.11][mac].
 All packages are available on our GitHub [releases][] page.
 
 All of these steps can be done automatically by the [`download.sh`][download] script.
```

Comment:
All `linux.md ` is for official distributives. Add community-supported platform here is misleading (Arch Linux is not on Releases page and not supported by `download.sh`).

We should to either change the structure of this file, or make a new `linux-community.md`.